File: aula3/estrutura.java
import java.util.Scanner;

public class Main {
    public static void main(string[]args){
        System.out.print("Digite a quantidade em estouqeu:");
        int quantidade = scanner.nextInt();


        //Usando a estrutura 'if'
        if (quantidade < 5){
            System.out.println("AVISO: O estoque deste produto está baixo!");
        }
        scanner.close();
        }    }
}


// usando a estrutura com senão
import java.util.Scanner;

public class Main {
    public static void main(string[]args){
        System.out.print("Digite a quantidade em estouqeu:");
        int quantidade = scanner.nextInt();


        //Usando a estrutura 'if'
        if (quantidade < 5){
            System.out.println("AVISO: O estoque deste produto está baixo!");
        }else{
            System.out.println("ESTOQUE OK");
        }
        scanner.close();
        }    }
}

// usando a estrutura if else else-if
import java.util.Scanner;

public class Main {
    public static void main(string[]args){
        System.out.print("Digite a quantidade em estouqeu:");
        int quantidade = scanner.nextInt();


        //Usando a estrutura 'if'
        if (quantidade < 1){
            System.out.println("Status: Em falta");
        }else if (quantidade < 5){
            System.out.println("Status: Alerta de estoque baixo");
        }
        else{
            System.out.println("Status: Estoque OK")
        }
        scanner.close();
        }    }
}

// switch


import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite o código de status (1-Disponível, 2=Reposição, 3=Descontinuado):");
        int codigo_status = scanner.nextInt();

        switch (codigo_status) {
            case 1:
                System.out.printIn("Status: Disponível");
                break;
            case 2: 
                System.out.PrintIn("Status: Em reposição");
                break;
            case 3:
                System.out.PrintIn("Status: Descontinuado");
                break;
            default:
                System.out.PrintIn("Código de status inválido");
        }
        scanner.close();
    }

File: aula3/estrutura.cpp
#include <iostream>
using namespace std;

int main(){
    int quantidade;
    cout << "Digite a quantidade em estoque:";
    cin >> quantidade;

    //Usando a estrutura 'if'
    if (quantidade < 5){
        cout << "Aviso: o estoque deste produto está baixo!" << endl;

    }
    return 0;
}

// usando a estrutura senão
#include <iostream>
using namespace std;

int main(){
    int quantidade;
    cout << "Digite a quantidade em estoque:";
    cin >> quantidade;

    //Usando a estrutura 'if'
    if (quantidade < 5){
        cout << "Aviso: o estoque deste produto está baixo!" << endl;

    } else{
        cout << "Estoque OK." << std::endl;
    }
    return 0;
}

// usando a estrutura if else-if
#include <iostream>
using namespace std;

int main(){
    int quantidade;
    cout << "Digite a quantidade em estoque:";
    cin >> quantidade;

    if (quantidade < 1){
        cout << "Status: Em falta.!" << endl;

    } else if (quantidade < 5 ){
        cout << "Status: Alerta de estoque baixo" << endl;
    }
    else{
        cout << "Status: Estoque OK"  << endl;
    }
    return 0;
}

// switch


#include <iostream>
using namespace std;

int main() {
    int codigo_status;
    cout << "Digite o código de status (1 = Disponível, 2 = Reposição, 3 = Descontinuado):";
    cin >> codigo_status;

    switch (codigo_status) {
        case 1: cout << "Status: Disponível." << endl;
            break;
        case 2: cout << "Status: Em Reposição." << endl;
            break;
        case 3: cout << "Status: Descontinuado." << endl;
            break;
        default: cout << "Código de status inválido." << endl;
    }
return 0;
}